client/src/index.js

```diff
@@ -9,7 +9,7 @@ import { BrowserRouter as Router, Route } from 'react-router-dom';
 const AppRouter = () => {
 	return (
 		<>
-			<Router>
+			<Router forceRefresh>
```

Comment:
Alex, could you please add more details about the reasoning here?

Reply:
This was due to a bug that was caused by me not closing the EventSource. I am not exactly sure why but the setNotification method would not work if a new EventSource was opened before closing the old one. I assumed that it was some bug with react and so I included the forceRefresh (which I guess closed the EventSource) but once I fixed the EventSource issue the whole issue went away so it is now removed.